src/components/AnnotationPopup/AnnotationPopup.js

```diff
@@ -186,6 +186,12 @@ const AnnotationPopup = () => {
     dispatch(actions.closeElement('annotationPopup'));
   };
 
+  const downloadFileAttachment = annot => {
+    // no need to check that annot is of type file annot as the check is done in the JSX
+    // trigger the annotationDoubleClicked event so that it will download the file
+    core.getAnnotationManager().trigger('annotationDoubleClicked', annot);
```

Comment:
do it this way, for now as it seems there is another fxn that does it this way.
the good part is, it reuses a lot of existing code.
bad part is, it looks weird